semantics/language/translation/compound-literal.k

```diff
@@ -58,15 +58,13 @@ module C-COMPOUND-LITERAL
                => allocObject(Base, T)
                ~> addToEnv(X, Base)
                ~> giveType(X, T)
-               ~> zero(X)
-               ~> Init
+               ~> allowInit(Init)
                ~> X
           ...</k>
           <curr-tu> Tu:String </curr-tu>
           <tu-id> Tu </tu-id>
           <next-static-loc> Base:SymBase => linc(Base) </next-static-loc>
-          <initializing> I:Int </initializing>
-          requires I >Int 0
+          <block-history> .List </block-history>
```

Comment:
This prevents block-scope compound literals from being allocated at translation-time. Global-scope literals can be allocated, as they're supposed to have static storage class and a lifetime for the whole program.